sky/skylet/providers/kubernetes/utils.py

```diff
@@ -47,22 +50,41 @@ def check_credentials(timeout: int = 3) -> Tuple[bool, Optional[str]]:
         return False, 'Failed to communicate with the cluster - timeout. ' \
                       'Check if your cluster is running and your network ' \
                       'is stable.'
+    except ValueError as e:
+        return False, str(e)
     except Exception as e:
         return False, f'An error occurred: {str(e)}'
```

Comment:
nit: A better way to print the exceptions will be `common_utils.format_exception(e)` so the type of the exception will b printed, making it easier to debug.